.cspell.json

```diff
@@ -12,7 +12,8 @@
     "extrafields",
     "gql.tada",
     "msword",
-    "Turborepo"
+    "Turborepo",
+    "persistor"
```

Comment:
we don't need this from what I can see.

Reply:
I tried by changing the name of the const but [`persistor` it's still present](https://github.com/bigcommerce/b2b-buyer-portal/pull/195#discussion_r1893073209) in `apps/storefront/src/react-setup.tsx` as prop and cspell throw an error as a not recognized word